virttest/test_setup.py

```diff
@@ -2082,8 +2104,15 @@ def cleanup(self):
         try:
             if os.path.exists(self.polkit_rules_path):
                 os.unlink(self.polkit_rules_path)
-            if os.path.exists(self.conf_backup_path):
-                os.rename(self.conf_backup_path, self.conf_path)
+            for drv in ['virtqemud', 'virtnetworkd', 'virtnodedevd',
+                        'virtnwfilterd', 'virtsecretd',
+                        'virtstoraged', 'virtinterfaced', 'libvirtd']:
+                conf_path = utils_config.get_conf_obj(drv).conf_path
+                conf_backup_path = conf_path + ".virttest.backup"
+                if os.path.exists(conf_backup_path):
+                    os.rename(conf_backup_path, conf_path)
+            libvirtd = utils_libvirtd.Libvirtd()
```

Comment:
`libvirtd` daemon(s) is(are) restarted in cleanup process, so do you need to clear the code about service restarting in env_process.postprocess()?